__tests__/ClusterManagement.js

```diff
@@ -47,6 +47,7 @@ describe('ClusterManagement', () => {
 
   /************ TESTS ************/
   it('creates a v5 cluster and redirect to details view', async () => {
+    jest.setTimeout(20000);
```

Comment:
This is a flaky test that fails 2/5 times. I increased its timeout so it won't bother us, for now